dashboard/app/models/contact_rollups_final.rb

```diff
@@ -15,4 +15,19 @@
 
 class ContactRollupsFinal < ApplicationRecord
   self.table_name = 'contact_rollups_final'
+
+  def self.overwrite_from_processed_table
+    ActiveRecord::Base.connection.truncate(table_name)
+    insert_from_processed_table
+  end
+
+  def self.insert_from_processed_table
+    overwrite_sql = <<~SQL
+      INSERT INTO contact_rollups_final
```

Comment:
Seems like we should use `table_name` here so there's one authoritative source for the table name.